### PR TITLE
set is_duplicate flag for both ids

### PR DESCRIPTION
**Pull request by kmagida110**

### CODE REVIEW

File: src/record_deduplication/dedupe.py [hunk 1]
```diff
@@ -131,11 +131,14 @@ def identify_duplicates(df, threshold, show_ranks_distribution=True, filename=No
     for id in ece_ids:
         if not id in dupes:
             new_ids[id] = i
-            is_dupes[id] = False
             i += 1
+            if id in dupes.values():
+                is_dupes[id] = True
+            else:
+                is_dupes[id] = False
```
Comment (bamader) [inline src/record_deduplication/dedupe.py:138]: How can this trigger? We're already inside a conditional for `id not in dupes` so what does this add?

Comment (kmagida110) [reply src/record_deduplication/dedupe.py:138]: The `not in dupes` is for the keys (1st item of the pair), the `id in dupes.values()` is for the values (2nd items in the pair)

**Comment by kmagida110** [comment]
@bamader This should add a `is_duplicate` flag to both pairs of a duplicate and not just the one that arbitrarily shows up second in the order.

**Review by bamader — APPROVED** [review]
Ohhhhh I'm grokking. So this lets us still properly use and increment the "new unique ID" (that we make solely for this script), but it marks _all_ records involved with that ID as duplicates rather than the unlucky record that happened to come second. Which is a good thing because we can't always assume that the first record encountered is the canonical "true" child.

**Review by JordanGuinn — APPROVED** [review]

**Pull request merged**